controllers/managedclusterForCGU_controller_test.go

```diff
@@ -264,6 +271,77 @@ func TestControllerReconciler(t *testing.T) {
 				}
 			},
 		},
+		{
+			name: "managed cluster is ready and partial found child policies are copied policies",
```

Comment:
Hmm... Not sure what you mean by "partial found child policies".

Reply:
Not accurate. Maybe => one found child policy is a copied policy